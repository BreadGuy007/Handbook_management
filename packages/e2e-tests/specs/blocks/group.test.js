/**
 * WordPress dependencies
 */
import {
	clickBlockAppender,
	searchForBlock,
	getEditedPostContent,
	createNewPost,
} from '@wordpress/e2e-test-utils';

describe( 'Group', () => {
	beforeEach( async () => {
		await createNewPost();
	} );

	it( 'can be created using the block inserter', async () => {
		await searchForBlock( 'Group' );
		await page.click( '.editor-block-list-item-group' );
		await page.keyboard.type( 'Group block' );

		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );

	it( 'can be created using the slash inserter', async () => {
		await clickBlockAppender();
		await page.keyboard.type( '/group' );
		await page.keyboard.press( 'Enter' );
		await page.keyboard.type( 'Group block' );

		expect( await getEditedPostContent() ).toMatchSnapshot();
	} );
} );
