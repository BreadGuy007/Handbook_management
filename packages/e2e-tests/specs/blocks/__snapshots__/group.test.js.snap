// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Group can be created using the block inserter 1`] = `
"<!-- wp:group -->
<div class=\\"wp-block-group\\"><!-- wp:paragraph -->
<p>Group block</p>
<!-- /wp:paragraph --></div>
<!-- /wp:group -->"
`;

exports[`Group can be created using the slash inserter 1`] = `
"<!-- wp:group -->
<div class=\\"wp-block-group\\"><!-- wp:paragraph -->
<p>Group block</p>
<!-- /wp:paragraph --></div>
<!-- /wp:group -->"
`;
